pkgs/servers/nfs-ganesha/default.nix

```diff
@@ -42,6 +42,10 @@ stdenv.mkDerivation rec {
     nfs-utils
   ];
 
+  postFixup = ''
+    patchelf --add-rpath $out/lib $out/bin/ganesha.nfsd
```

Comment:
@thefloweringash It appears that since https://github.com/NixOS/nixpkgs/pull/162447 the binary does not contain its `lib` directory anymore in the `rpath`. As a result the binary is broken. Do you know a better method to fix this than calling `patchelf` manually (as I am doing right now)?

Reply:
Looks like it is cmake mangling the RPATH:
```
Install the project...
-- Install configuration: "Release"
-- Installing: /nix/store/lgaapc69qiksccsbmvjn95q2fb5s8f12-nfs-ganesha-5.3/etc/ganesha/ganesha.conf
-- Installing: /nix/store/lgaapc69qiksccsbmvjn95q2fb5s8f12-nfs-ganesha-5.3/bin/ganesha.nfsd
-- Set runtime path of "/nix/store/lgaapc69qiksccsbmvjn95q2fb5s8f12-nfs-ganesha-5.3/bin/ganesha.nfsd" to ""
-- Installing: /nix/store/lgaapc69qiksccsbmvjn95q2fb5s8f12-nfs-ganesha-5.3/lib64/ganesha/libfsalnull.so
-- Set runtime path of "/nix/store/lgaapc69qiksccsbmvjn95q2fb5s8f12-nfs-ganesha-5.3/lib64/ganesha/libfsalnull.so" to ""
...
```

Reply:
Thanks for the tip. Any idea how to tell cmake to include it again?

Reply:
You may try grepping for `RPATH` in the cmake files to see if any relevant options are set.

Reply:
@NickCao There is no obvious rpath issue in the cmake files of the package. I still think that the above mentioned PR caused the problem. The cmake files are pretty vanilla and it should set the rpath automatically.

If no one can come up with another solution, I would suggest to use the proposed manual `patchelf` solution to get the package back into a working state.